src/main/resources/templates/home.html

```diff
@@ -4,7 +4,7 @@
 <head>
 	<meta charset="ISO-8859-1"/>
 	<link rel="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
-	<link rel="stylesheet" type="text/css" href="../static/stylesheet.css" />
+	<link rel="stylesheet" type="text/css" href="stylesheet.css" />

```

Comment:
Your stylesheet path didn't resolve, if you see issues with your page open your developer panel in the browser. In most browsers the keybind is ctrl + shift + i. If you hit the network tab and refresh the page (ctrl + r) you will see the requests that the page made while loading. This is incredibly useful for debugging problems with your API's or even with stuff like this. Once I saw that your path wasn't working, it was pretty easy to fix.

Reply:
Very helpful tip!